src/containers/ModalAuthorization.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
```

Comment:
not really what I meant. It wasn't your task, so it's fine and you just improvised, so it's fine to merge as it is, but it's a bit dumb to create <ModalSomething> component for each content that we would put in Modal. Right way to do that is to create reusable and abstract Modal component, that renders Button with text from props, and internally adding onClick handler to this button to open modal with content from props.children.

it's not your task, so doesn't matter

Reply:
I think I did what you originally wanted. But need confirmation from you.